plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHTunnelConfiguratorUI.java

```diff
@@ -183,12 +191,6 @@ public void widgetSelected(SelectionEvent e) {
             setNumberEditStyles(tunnelTimeout);
         }
 
-        {
-            Composite sc = new Composite(composite, SWT.NONE);
-            sc.setLayout(new GridLayout());
-            sc.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true));
-        }
-
```

Comment:
It was a spacer that forced the link widget to be at the very bottom. What's the point of this change?